easybuild/easyconfigs/q/QUDA/QUDA-1.1.0-foss-2022a-CUDA-11.7.0.eb

```
@@ -0,0 +1,40 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = 'CMakeMake'
+
+name = 'QUDA'
+version = '1.1.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://github.com/lattice/quda'
+description = """QUDA is a library for performing calculations in lattice QCD
+ on graphics processing units (GPUs), leveraging NVIDIA's CUDA platform."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+github_account = 'lattice'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['b4f635c993275010780ea09d8e593e0713a6ca1af1db6cc86c64518714fcc745']
+
+builddependencies = [
+    ('CMake', '3.23.1'),
+    ('Python', '3.10.4'),
+]
+
+dependencies = [
+    ('CUDA', '11.7.0', '', True),
```

Please use `SYSTEM` instead of `True` (since the use of `True` is deprecated); see also https://docs.easybuild.io/en/latest/Writing_easyconfig_files.html#specifying-dependencies-using-system-toolchain

```suggestion
    ('CUDA', '11.7.0', '', SYSTEM),